.husky/.gitignore

```diff
@@ -0,0 +1 @@
+_
```

Comment:
could you explain?

Reply:
This is done by the NPM package [`husky`](https://npmjs.com/package/husky), not by me. This is a new thing in husky@6.